tests/assertion-removal.test.ts

```diff
@@ -1,4 +1,4 @@
-import assert from 'assert';
+import assert from 'node:assert/strict';
```

Comment:
The change from `assert` to `node:assert/strict` is harmless for integration tests because they're not being processed by sku.